bundle/.flatpak-builder/ccache/disabled/ccache.conf

```diff
@@ -0,0 +1 @@
+disable = true
```

Comment:
why do you need this?

Reply:
I don't. These must be some files that my local build left there. I will delete it.